hw/ip/otbn/dv/rig/rig/gens/bad_deep_loop.py

```diff
@@ -51,6 +52,8 @@ def _pick_bodysize(self,
         # 4096, giving a maximum address of 16KiB, which happens to equal the
         # size of IMEM, we know it's going to be possible. Try to pick 50
         # times, which should fail with probability at most 1/4^50 = 1/2^100.
+        # If the IMEM is greater than 16KB we attempt to find a suitable
+        # bodysize but return None if one is not found.
```

Comment:
nit: This sounds like the code checks the IMEM size and does a different thing in the > 16kB case, but it just does the same attempt loop each time and allows itself to fail because IMEM might be > 16kB. I'd suggest rephrasing to avoid confusion, e.g.:
```suggestion
        # 4096, it is always possible if IMEM size is 16KiB. Try to pick 50
        # times, which should fail with probability at most 1/4^50 = 1/2^100.
        # If the IMEM is greater than 16KB, it might not be possible to find a
        # body size that fits these criteria, so we might fail and return None.
```